HousingSearchListener/V1/UseCase/IndexCreatePersonUseCase.cs

```diff
@@ -9,13 +9,15 @@
 
 namespace HousingSearchListener.V1.UseCase
 {
-    public class IndexPersonUseCase : IIndexPersonUseCase
+    public class IndexCreatePersonUseCase : IIndexCreatePersonUseCase
     {
         private readonly IEsGateway _esGateway;
         private readonly IPersonApiGateway _personApiGateway;
         private readonly IESEntityFactory _esPersonFactory;
 
-        public IndexPersonUseCase(IEsGateway esGateway, IPersonApiGateway personApiGateway,
+        public Person Person { get; set; }
```

Comment:
would remove this property and get the person again in the `IndexUpdatePersonUseCase` use case